src/protocol/requests/createAcls/v1/response.js

```diff
@@ -0,0 +1,14 @@
+const { decode, parse } = require('../v0/response')
+
+/**
+ * CreateAcls Response (Version: 1) => throttle_time_ms [creation_responses]
+ *   throttle_time_ms => INT32
+ *   creation_responses => error_code error_message
+ *     error_code => INT16
+ *     error_message => NULLABLE_STRING
+ */
+
+module.exports = {
+  decode,
```

Comment:
v1 of all the `*acl` APIs return the responses before throttling, meaning that we should support client side throttling. That's already been implemented on the network layer, so all you have to do on the protocol level is to copy the `throttleTime` property to a `clientSideThrottleTime` property on the decoded object and the rest is handled automatically (see https://github.com/tulios/kafkajs/blob/900c60a7473d16b7f678be4284ed95f21587b789/src/protocol/requests/heartbeat/v2/response.js#L11-L18 for example)